docs/claimantmodel/CoreModel.md

```diff
@@ -0,0 +1,44 @@
+# Claimant Model
+
+This paper introduces an intuitive, standalone model for a common scenario where one party needs to perform an action which is reasonable only if a claim made by another party is true. The Claimant Model allows formally verifiable constructs to be bound to a messy reality.
+
+## Model
+
+*This model describes roles, not actors. In a concrete deployment of this model, the same actor may play multiple roles, or multiple actors may need to work together to cover a role.*
+
+There is a **Claimant** that makes a **Claim** that is relied upon by a **Believer** as a precondition to take an action they would not have taken if the claim was false. Claims are represented by signed **Statements**. The veracity of a Claim can be verified by a **Claim Verifier**, who will notify a **Claim Arbiter** of any false Claims.
+
+## Example: Certification Claims
+* Claim<sup>CERT</sup>: (I, ${CA}, am authorized to certify $pubKey for $domain)
```

Comment:
Could try using MD definitions or a table here to tidy?

Reply:
I've used a definition list. I'm not totally convinced it's better but I like compactness.